core/integer/exponent_spec.rb

```diff
@@ -0,0 +1,7 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+require File.expand_path('../shared/exponent', __FILE__)
+
+describe "Integer#**" do
+  it_behaves_like :integer_exponent, :**
+end
```

Comment:
Currently there is Fixnum#** and Bignum#** specs, but it sounds best to follow the 2.4 unified Integer structure.
I filed an issue to address this in general: #578.